docs/designers-developers/developers/themes/theme-support.md

```diff
@@ -8,7 +8,6 @@ There are a few new concepts to consider when building themes:
 - **Editor Text Size Palette** - A default set of sizes is provided, but themes can register their own and optionally lock users into picking from preselected sizes.
 - **Responsive Embeds** - Themes must opt-in to responsive embeds.
 - **Frontend & Editor Styles** - To get the most out of blocks, theme authors will want to make sure Core styles look good and opt-in, or write their own styles to best fit their theme.
-- **Dark Mode** - If a Theme is a Dark Theme with a dark background containing light text, the theme author can opt-in to the Dark Mode.
```

Comment:
Love that we don't use this phrase :) It can be dark mode, or it can just be a dark background! 👌